packages/calling/src/CallingClient/line/index.ts

```diff
@@ -103,6 +106,7 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
     const serviceData = serviceDataConfig?.indicator
       ? serviceDataConfig
       : {indicator: ServiceIndicator.CALLING, domain: ''};
+    this.serviceData = serviceData;
```

Comment:
One way to avoid creating a new private variable is to move the serviceData declaration under the constructor outside and make it a global variable with a default value of Calling. And within the constructor it will get updated with the value received in the parameter